webui/react/src/pages/Settings/GroupManagement.tsx

```diff
@@ -27,7 +27,10 @@ import handleError from 'utils/error';
 import { useObservable } from 'utils/observable';
 
 import css from './GroupManagement.module.scss';
-import settingsConfig, { DEFAULT_COLUMN_WIDTHS } from './GroupManagement.settings';
+import settingsConfig, {
+  DEFAULT_COLUMN_WIDTHS,
+  DEFAULT_COLUMNS,
```

Comment:
just my thought, no need to change anything
feel like we dont need `DEFAULT_COLUMNS` because its always fixed and no way to change columns in the page iirc. However,

Reply:
It's a good point that we can't change columns here anyway. Will just merge this for now for the fix but we can revisit.